app/gui2/src/components/CodeEditor.vue

```diff
@@ -306,3 +306,4 @@ const editorStyle = computed(() => {
   border-radius: 3px 0 0 3px;
 }
 </style>
+@/util/data/result @/util/data/iterable
```

Comment:
There are weird trailing module names everywhere. Probably due to automatic refactor screwing something up. Please check all modified vue files for them, as it is present in almost every one.